src/scss/components/application/_application.scss

```diff
@@ -19,6 +19,7 @@
   width: $width;
 }
 .application {
+  overflow: hidden;
```

Comment:
It seems this doesn't fix the x-overflow, there's side scrolling on smaller screen sizes :/

Reply:
the width unit for the footer should scale with the window/screen width, like vw